website/src/app/(home)/widgets/page.css

```
@@ -0,0 +1,183 @@
+    background-image:
+      linear-gradient(to bottom, white, transparent),
+      url(/hero-tel-input/assets/high-contrast.png);
+  }
+}
+@media screen and (max-width: 1200px) {
+  #highlighting-card {
+    aspect-ratio: auto;
+  }
+  .feat-darkmode h3 {
+    font-size: 4vw;
+    font-size: min(48px, max(4vw, 30px));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)